static/app/views/settings/project/server-side-sampling/rule.tsx

```diff
@@ -117,16 +124,33 @@ export function Rule({
           onFinish={() => {
             // TODO(sampling): activate the rule
           }}
-          // TODO(sampling): disable if sdks are not yet updated
-          disabled={true || !isUniform}
+          disabled={!canActivate || !isUniform}
         >
-          <ActiveToggle
-            inline={false}
-            hideControlState
-            aria-label={rule.active ? t('Deactivate Rule') : t('Activate Rule')}
-            onClick={onActivate}
-            name="active"
-          />
+          <Tooltip
+            disabled={canActivate}
+            title={
+              !canActivate
+                ? upgradeSdkForProjects.length === 1
+                  ? t(
+                      'To enable the rule, the sdk version for the project %s have to be updated',
+                      upgradeSdkForProjects[0]
+                    )
+                  : t(
+                      'To enable the rule, the sdk version for the projects %s have to be updated',
+                      upgradeSdkForProjects.join(', ')
```

Comment:
Should we use `tn` function instead? We probably don't even need to list the projects in the tooltip and can display something a bit more shorter because the big alert with more details will be visible on that page.

Reply:
any ideas for the copy?